pkgs/development/python-modules/fava-investor/default.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  stdenv,
+  beancount,
+  click,
+  click-aliases,
+  fava,
+  packaging,
+  pytestCheckHook,
+  python-dateutil,
+  setuptools,
+  setuptools-scm,
+  tabulate,
+  yfinance,
+}:
+buildPythonPackage rec {
+  pname = "fava_investor";
```

```suggestion
  pname = "fava-investor";
```
see the language specific docs for python